docs/proposals/rename-group.md

```diff
@@ -0,0 +1,207 @@
+Add a wrap client for controller-runtime client to automatically convert a group of LogicalVolume resources.
+Change each controller and other components to use this wrapped client.
+
+### Things to be done by users of topolvm.cybozu.com for upgrading the helm chart
+
+Since this change was released, users of TopoLVM with the group name `topolvm.cybozu.com` will need to take the following actions when upgrading the helm chart.
+
+- Set the `.Values.useLegacy `variable to` true` in `values.yaml` in the helm chart
+- If there is a part of the `values.yaml` in the helm chart that uses the string `topolvm.cybozu.com`, change it to `topolvm.io`
+
+If you upgrade the helm chart without taking above actions, it is possible that TopoLVM related resources will be deleted.
+
+### Migrate from topolvm.cybozu.com to topolvm.io
+
+If you are already using TopoLVM with the group name `topolvm.cybozu.com` and want to migrate the group name to `topolvm.io` after the release of this change, you can manually change the data by the following method.
+
+**You should backup Kubernetes resources (PV, PVC, LogicalVolume CR) and lvm that data managed by TopoLVM because the data could be lost when the migration is failed.**
+**Also, please verify in advance whether the migration will succeed without any problems before executing the migration.**
+**Since the migration procedure is complicated, please consider using TopoLVM as topolvm.cybozu.com without migration.**
+
+1. Avoid booting pods using TopoLVM volumes during migration.
+1. Temporarily stop the following pods:
+   - topolvm-controller
+   - topolvm-node
+   - topolvm-scheduler
+1. Remove legacy TopoLVM resources without LogicalVolume CRD
+1. Manually install the new TopoLVM without topolvm-node to another namespace
+1. Update a scheduler config and restart a kube-scheduler
+1. Perform the migration work for each resource as mentioned in the chapter `Changes to topolvm.io`.
+   - Please migrate while confirming that the pod using the updated resource (e.g. Persistent Volume) continues to operate without problems.
+1. Start topolvm-node after resource migration
+
+#### Manual migration procedure
+
+##### LogicalVolume
+
+1. Run command like `kubectl get logicalvolumes.topolvm.cybozu.com <name> -o yaml > logicalvolumes.yaml` and get current LogicalVolume data.
+1. Edit a `logicalvolumes.yaml` file like follows:
+   * Remove unnecessary metadata fields like a `.metadata.creationTimestamp`
+   * Edit `.apiVersion` value to `topolvm.io/v1`
+   * If there is a `topolvm.cybozu.com/resize-requested-at` field in `.metadata.annotations`, rewrite it to `topolvm.io/resize-requested-at`
+   * If there is `topolvm.cybozu.com/logicalvolume` in `.metadata.finalizers`, delete it and add `topolvm.io/logicalvolume`
+1. Run command like `kubectl apply -f logicalvolumes.yaml` and create a migrated LogicalVolume
+1. Set the `.status` field to the value from `logicalvolumes.yaml` with `kubectl edit logicalvolumes.topolvm.io <name>`
+
```

Comment:
I suspect that this is not an intended procedure.
I thought that the following procedure is correct.

1. deploy a new topolvm (topolvm.io) (at this point, old and new topolvm are working simultaneously.)
3. manually copy all data from PVCs of old topolvm (topolvm.cybozu.com) to new one (topolvm.io).
4. delete old topolvm (topolvm.cybozu.com)

@llamerada-jp Is my understanding correct?